openerp/addons/base/ir/ir_attachment.py

```diff
@@ -427,8 +430,26 @@ def create(self, cr, uid, values, context=None):
             values.pop(field, False)
         values = self._check_contents(cr, uid, values, context=context)
         self.check(cr, uid, [], mode='write', context=context, values=values)
+        if 'url' in values:
```

Comment:
Ah, of course, it creates an attachment but without url. Sorry for my confusion.